tools/integration-tests/tester/framework/docker.go

```diff
@@ -65,6 +65,7 @@ func (d *DockerContainer) CreateGoShimmerEntryNode(name string, seed string) err
 			fmt.Sprintf("--node.disablePlugins=%s", disabledPluginsEntryNode),
 			"--autopeering.entryNodes=",
 			fmt.Sprintf("--autopeering.seed=base58:%s", seed),
+			fmt.Sprintf("--database.forceCacheTime=%d", noCache),
```

Comment:
Why do you have a separate configuration for EntryNode? 
Why don't just do as for peer:
```go
			fmt.Sprintf("--database.forceCacheTime=%s", config.ForceCacheTime)
```
With `ForceCacheTime=0`?

Reply:
I noticed that there is a current pattern in the code where entry node has different hard-coded configs

I didn't want to break this pattern, even though I don't really understand why it exists...
So before I change anyhting can you tell me why all the other peer configs are not passed to entry node?

Reply:
Ah okay, I the entry node doesn't work as a general Goshimmer node, most of the plugins are disabled. 
As you can see in the list `disabledPluginsEntryNode` the `messagelayer` plugin is disabled, so the database won't be used at all. You can just remove the cache configuration at all for an entry node.